### ISSUE TITLE
plotProfile ignores xlim argument, for "non-standard" fields

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1009]
There are lots of individual `lim` arguments, e.g. `Slim`, `Tlim`, `plim` etc, however if you want to control the axis limits for a "non-standard" field in a ctd object there is no way to do it. Example:
```r
library(oce)
data(ctd)
ctd <- oceSetData(ctd, 'junk', rnorm(ctd[['pressure']]), list(unit=expression(JNK), scale=''))
plotProfile(ctd, xtype='junk', xlim=c(-5, 5))
```
<img width="812" alt="screen shot 2016-06-30 at 23 36 53" src="https://cloud.githubusercontent.com/assets/233584/16511175/8e51d468-3f1b-11e6-9771-6cd50784ae02.png">

**Comment by dankelley** [comment]
I think this is fixed in `develop` commit 558ba7d14dedaa692e5a5d1d815332f19387e763

I made a test suite for it, with results as below ... I didn't put purple labels telling what to expect but I think you can tell by inspection whether this worked.
```R
library(oce)
data(ctd)
ctd <- oceSetData(ctd, 'junk', rnorm(ctd[['pressure']]), list(unit=expression(JNK), scale=''))
if (!interactive()) png("1009a.png")
par(mfrow=c(2,2))
plotProfile(ctd, xtype='junk')
plotProfile(ctd, xtype='junk', xlim=c(-5, 5))
plotProfile(ctd, xtype='temperature')
plotProfile(ctd, xtype='temperature', Tlim=c(0, 30))
if (!interactive()) dev.off()
```
![1009a](https://cloud.githubusercontent.com/assets/99469/16519519/ea9e1b84-3f5f-11e6-8c32-ddd550eb81cd.png)

**Comment by richardsc** [comment]
Great, thanks!
